pkg/http/util.go

```diff
@@ -49,7 +49,7 @@ func ConvertEventToBytes(event interface{}, contentType string) ([]byte, error)
 	} else if contribContentType.IsJSONContentType(contentType) || contribContentType.IsCloudEventContentType(contentType) {
 		dBytes, err := json.Marshal(event)
 		if err != nil {
-			return []byte{}, errors.Wrapf(err, errContentTypeMismatch.Error())
+			return []byte{}, fmt.Errorf("%s: %w", errContentTypeMismatch.Error(), err)
```

Comment:
I think here it should be the opposite?

`fmt.Errorf("%w: %v", errContentTypeMismatch.Error(), err)`

Because this way you can do `errors.Is(err, errContentTypeMismatch)`? Unless that's not necessary?

Reply:
Please see the test that passes with the format ....

Reply:
Oh,ok